### PR TITLE
Added VCF constants for Mutect2 filters

### PR DESCRIPTION
**Pull request by takutosato**
Triallelic site, strand artifact and clustered read position filters.

### CODE REVIEW

**Comment by takutosato** [comment]
@davidbenjamin will you review this? It's a very small change to add some Mutect2 filters to GATK4.

**Comment by davidbenjamin** [comment]
@takutosato Of course.

File: src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java [hunk 1]
```diff
@@ -137,6 +137,9 @@ their names (or descriptions) depend on some threshold.  Those filters are not i
     public static final String PON_FILTER_NAME =                              "panel_of_normals"; //M2
     public static final String STR_CONTRACTION_FILTER_NAME =                  "str_contraction"; //M2
     public static final String TUMOR_LOD_FILTER_NAME =                        "t_lod_fstar"; //M2
+    public static final String TRIALLELIC_SITE_FILTER_NAME =                  "triallelic_site"; //M2
```
Comment (davidbenjamin) [inline src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java]: While you're at it, could you also move the following from `Mutect2Engine` (or rather copy it for now and delete from `Mutect2Engine` later once gatk-protected is pointed to the new gatk snapshot):
```java
public static final String TUMOR_SB_POWER_FWD_KEY =             "TUMOR_SB_POWER_FWD";
public static final String TLOD_FWD_KEY =                       "TLOD_FWD";
public static final String TLOD_REV_KEY =                       "TLOD_REV";
public static final String TUMOR_SB_POWER_REV_KEY =             "TUMOR_SB_POWER_REV";
public static final String MEDIAN_LEFT_OFFSET_KEY =              "MEDIAN_LEFT_OFFSET";
public static final String MEDIAN_RIGHT_OFFSET_KEY =             "MEDIAN_RIGHT_OFFSET";
public static final String MAD_MEDIAN_LEFT_OFFSET_KEY =          "MAD_LEFT_OFFSET";
public static final String MAD_MEDIAN_RIGHT_OFFSET_KEY =         "MAD_RIGHT_OFFSET";
```

**Comment by davidbenjamin** [comment]
@takutosato back to you

**Comment by takutosato** [comment]
Added the keys @davidbenjamin referenced.

**Comment by davidbenjamin** [comment]
@takutosato Looks good.  The Travis fail was obviously not your fault and I restarted it.  If it passes you can squash and merge.

**Pull request merged**